packages/_site/src/pages/components/switch/index.tsx

```diff
@@ -119,6 +121,163 @@ export default function SwitchExample() {
+
+    return (
+		<Switch
+			checked={switchChecked}
+			onCheckedChange={(e) => {
+				setSwitchChecked(e);
+			}}
+            size={SwitchSizes.LARGE}
+		/>
+	);
+}
+`}
+                                />
+                            </div>
+                        </div>
+                        <div>
+                            <ExampleBlock
+                                title="Disabled"
+                                description="When the switch selection has no need to be changable, set the state to disabled."
+                            />
+                            <div {...stylex.props(styles.codeWrapper)}>
+                                <PreviewBlock>
+                                    <Switch
+                                        checked={false}
+                                        onCheckedChange={(e) => {
+                                            setSwitchChecked(e);
+                                        }}
+                                        disabled
+                                    />
+                                </PreviewBlock>
+
+                                <CodeBlock
+                                    language="tsx"
+                                    code={`import React, { useState } from 'react';
+
+import { Switch } from "@controlkit/switch";
+
+export default function SwitchExample() {
+	const [switchChecked, setSwitchChecked] = useState<boolean>(false);
+
+    return (
+		<Switch
+			checked={switchChecked}
+			onCheckedChange={(e) => {
+				setSwitchChecked(e);
+			}}
+            disabled
+		/>
+	);
+}
+`}
+                                />
+                            </div>
+                        </div>
+
+                        <div>
+                            <ExampleBlock
+                                title="Toggled and Disabled"
+                                description="Disabled switches can also be toggled on."
+                            />
+                            <div {...stylex.props(styles.codeWrapper)}>
+                                <PreviewBlock>
+                                    <Switch
+                                        checked={true}
+                                        onCheckedChange={(e) => {
+                                            setSwitchChecked(e);
+                                        }}
+                                        disabled
+                                    />
+                                </PreviewBlock>
+
+                                <CodeBlock
+                                    language="tsx"
+                                    code={`import React, { useState } from 'react';
+
+import { Switch } from "@controlkit/switch";
+
+export default function SwitchExample() {
+	const [switchChecked, setSwitchChecked] = useState<boolean>(false);
+
+    return (
+		<Switch
+			checked={switchChecked}
+			onCheckedChange={(e) => {
+				setSwitchChecked(e);
+			}}
+		/>
+	);
+}
+`}
```

Comment:
Remove this, its redundant since we already have the disabled prop example just above it.